### DISCUSSION TITLE
Where is docker root dir located on host?

### DISCUSSION THREAD

**Discussion by arjinfo** [abiosoft/colima#431] [Q&A]
I'm unable to find `/var/lib/docker` on my m1 mac. If I ssh into colima I can find it on the alpine instance, not the ubuntu. I'd like to run commands like `cat $(docker inspect python3 | jq '.[0]' | jq -r .HostsPath)`.

**Comment by abiosoft** [comment]
Are you running Colima on macOS or Linux?

**Reply by arjinfo** [reply]
macOS

**Comment by abiosoft** [comment]
If I may ask, what are you trying to achieve? 

Docker does not run natively on macOS, rather it runs in a virtual machine.
